lodash.src.js

```diff
@@ -10610,6 +10610,40 @@
     }
 
     /**
+     * Replace all occurrences of the pattern with the replacement string.
+     *
+     * @static
+     * @memberOf _
+     * @category String
+     * @param {string} [string] The subject to change.
+     * @param {string} [pattern] The pattern to replace.
+     * @param {string} [replacement] The replacement value.
+     * @returns {string} Returns the subject with replaced pattern.
+     * @example
+     *
+     * _.replaceAll('a/b/c', '/', '-');
+     * // => 'a-b-c'
+     *
+     * _.replaceAll('a/b/c', '/', '//');
+     * // => 'a//b//c'
+     *
+     * _.replaceAll('aabbaabbaacc', 'aa', 'd');
+     * // => 'dbbdbbdcc'
+     */
+    function replaceAll(string, pattern, replacement) {
+      string = baseToString(string);
+      pattern = baseToString(pattern);
+
+      if (!pattern) {
```

Comment:
this doesn't seem strictly necessary, replace all(str, '', seperator) seems valid